src/pybind/rados/rados.pyx

```diff
@@ -1356,6 +1356,7 @@ Rados object in state %s." % self.state)
         io.io = ioctx
         return io
 
+    @requires(('cmd': str), ('inbuf': bytes), ('timeout', opt(int)), ('target', opt(str)))
```

Comment:
should change the docstring if inbuf is supposed to be bytes

Reply:
inbuf should be optional too

Reply:
@jdurgin the types of parameters in docstring were removed in another commit in this changeset. as it will be rendered by`sphinx_autodoc_typehints`  thanks to https://github.com/ceph/ceph/blob/075cc0ffd66b0523ebd22143d38765dba903df55/doc/conf.py#L58

Reply:
I see, much nicer!